torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -117,6 +117,7 @@ struct KinetoThreadLocalState : public ProfilerThreadLocalState {
   void reportMemoryUsage(
       void* /* unused */,
       int64_t alloc_size,
```

Comment:
I doubt that the downstream memory statistics will be influenced if "alloc_size"'s meaning is changed to absolute value. For example, the code [here](https://github.com/pytorch/pytorch/blob/0d0f4712b409e36d92595bb19140abfbd42cce86/torch/autograd/profiler.py#L1167) will get wrong result of adding many absolute sizes together, instead of originally adding many relative sizes to calculate a absolute size.

Reply:
Updated